.circleci/continue_config.yml

```diff
@@ -194,21 +194,47 @@ jobs:
       - checkout
       - restore_cache:
           key: ci-cache-<< parameters.python-version >>-<< parameters.extras >>-{{ checksum "pyproject.toml" }}
-      - run:
-          name: Run << parameters.pytest-group >> integration tests
-          command: |
-            export KOLENA_TOKEN=${KOLENA_TOKEN}
-            export KOLENA_CLIENT_BASE_URL=${KOLENA_CLIENT_BASE_URL}
-            TEST_GROUP="<< parameters.pytest-group >>"
-            if [ "$TEST_GROUP" = "misc" ]; then
-              TESTFILES=$(circleci tests glob tests/integration/test_*.py |
-                circleci tests split --split-by=timings --timings-type=filename)
-            else
-              TESTFILES=$(find tests/integration/$TEST_GROUP -name "test_*.py" |
-                circleci tests split --split-by=timings --timings-type=filename)
-            fi
-            poetry run pytest -vv --durations=10 --cov=kolena --cov-branch -o junit_family=legacy \
-              --junitxml=test-results/result.xml $TESTFILES
+      - when:
+          condition:
+            and:
+              - equal:
+                  - "none"
+                  - << parameters.extras >>
+              - not_equal:
+                  - _experimental
+                  - << parameters.pytest-group >>
+          steps:
+            - run:
+                name: Run << parameters.pytest-group >> integration tests
+                command: |
+                  export KOLENA_TOKEN=${KOLENA_TOKEN}
+                  export KOLENA_CLIENT_BASE_URL=${KOLENA_CLIENT_BASE_URL}
+                  TEST_GROUP="<< parameters.pytest-group >>"
+                  if [ "$TEST_GROUP" = "misc" ]; then
+                    TESTFILES=$(circleci tests glob tests/integration/test_*.py |
+                      circleci tests split --split-by=timings --timings-type=filename)
+                  else
+                    TESTFILES=$(find tests/integration/$TEST_GROUP -name "test_*.py" |
+                      circleci tests split --split-by=timings --timings-type=filename)
+                  fi
+                  poetry run pytest -vv --durations=10 --cov=kolena --cov-branch -o junit_family=legacy \
+                    --junitxml=test-results/result.xml $TESTFILES
+      - when:
+          condition:
+            and:
+              - equal:
+                  - "metrics"
+                  - << parameters.extras >>
+              - equal:
+                  - _experimental
+                  - << parameters.pytest-group >>
```

Comment:
Nit: do we need this condition, given that we are also using `-m 'metrics'` marker selection?

Reply:
I believe so. Correct me if I am wrong, but we need `<< parameters.extras >> == "metrics"` so that the restored cache is suitable